doc/examples/segmentation/plot_rolling_ball.py

```diff
@@ -0,0 +1,270 @@
+# Image Datatypes
+# ---------------
+#
+# ``rolling_ball`` can handle datatypes other than `np.uint8`. You can
+# pass them into the function in the same way.
+
+image = data.coins().astype(np.uint16)
+
+background = morphology.rolling_ball(image, radius=70.5)
+plot_result(image, background)
+plt.show()
+
+######################################################################
+# However, you need to be careful if you use floating point images
+# that have been normalized to ``[0, 1]``. In this case the ball will
+# be much larger than the image intensity, which can lead to
+# unexpected results.
+
+image = util.img_as_float(data.coins())
+
+background = morphology.rolling_ball(image, radius=70.5)
+plot_result(image, background)
+plt.show()
+
+######################################################################
+# Because ``radius=70.5`` is much larger than the maximum intensity of
+# the image, the effective kernel size is reduced significantly, i.e.,
+# only a small cap (approximately ``radius=10``) of the ball is rolled
+# around in the image. You can find a reproduction of this strange
+# effect in the ``rolling_ellipsoid`` section below.
+#
+# To get the expected result, you need to use the more flexible
+# ``rolling_ellipsoid`` function. It works just like ``rolling_ball``;
+# however, it gives you more control over the affected area and
+# strength of the algorithm. In particular, it has different
+# parameters for the spatial dimensions and the intensity dimension of
+# the image.
+
+image = util.img_as_float(data.coins())
+
+normalized_radius = 70.5 / 255
+background = morphology.rolling_ellipsoid(
+    image,
+    kernel_shape=(70.5 * 2, 70.5 * 2),
+    intensity_vertex=normalized_radius * 2
+)
+plot_result(image, background)
+plt.show()
+
+######################################################################
+# Rolling Ellipsoid
+# -----------------
+#
+# In ``rolling_ellipsoid`` you are specifying an ellipsoid instead of
+# a ball/sphere - sidenote: a ball is a special case of an ellipsoid
+# where each vertex has the same length. To fully specify an ellipsoid
+# in 3D, you need to supply three parameters. Two for the two spatial
+# dimensions of the image (via ``kernel_shape``), and one for the
+# intensity dimension (via ``intensity_vertex``).
+#
+# As mentioned above, a sphere is just a special ellipsoid, and hence
+# you can get the same behavior as ``rolling_ball`` if you set all
+# vertices to the respective values. In fact, ``rolling_ball``
+# internally calls ``rolling_spheroid`` in the way shown below.
+#
+# Note: The radius is equal to the length of a semi-vertex of a
+# sphere, which is *half* a full vertex. Hence, you need to multiply
+# the inputs by two if you want to get the same result as
+# ``rolling_ball``.
+
+image = data.coins()
+
+background = morphology.rolling_ellipsoid(
+    image,
+    kernel_shape=(70.5 * 2, 70.5 * 2),
+    intensity_vertex=70.5 * 2
```

Comment:
If you interpret a circle as a special case of an ellipsis, then the radius is equal to a semi vertex. A semi vertex is half a full vertex and hence `*2`.

For `kernel_shape` it is necessary, because we need to fit the full circle. For `intensity_vertex` it is possible to replace it with `intensity_semi_vertex` and not multiply, but I fear that might be confusing. I had to look up the term "semi vertex" and I wouldn't expect people to intuitively understand what it is. It's also slightly inconsistent to specify the vertex length for space dimensions and semi vertex length for the intensity, but maybe it's just me?